app/src/main/java/org/dhis2/uicomponents/map/layer/EnrollmentMapLayer.kt

```diff
@@ -0,0 +1,98 @@
+package org.dhis2.uicomponents.map.layer
+
+import com.mapbox.mapboxsdk.maps.Style
+import com.mapbox.mapboxsdk.style.expressions.Expression
+import com.mapbox.mapboxsdk.style.layers.FillLayer
+import com.mapbox.mapboxsdk.style.layers.Layer
+import com.mapbox.mapboxsdk.style.layers.LineLayer
+import com.mapbox.mapboxsdk.style.layers.Property
+import com.mapbox.mapboxsdk.style.layers.PropertyFactory
+import com.mapbox.mapboxsdk.style.layers.SymbolLayer
+import org.dhis2.uicomponents.map.TeiMapManager.Companion.ENROLLMENT_SOURCE_ID
+import org.dhis2.utils.ColorUtils
+import org.hisp.dhis.android.core.common.FeatureType
+
+class EnrollmentMapLayer(
```

Comment:
Inside the folder layer, I would create a folder called type.

you will have layer > types > heatmap, enrollment, satellite, tie etc...
and the in the layer > Maplayer interface and dialog

What do you think???